binderhub/static/js/pages/NotFoundPage.jsx

```diff
@@ -0,0 +1,26 @@
+export function NotFoundPage() {
+  return (
+    <>
+      <div className="container text-center jumbotron">
+        <h1>404: Not Found</h1>
+
+        <p className="lead">
+          Note: Some errors disappear by refreshing the page.
```

Comment:
I don't think this is a helpful hint for 404. While some errors go away with a refresh, 404 isn't likely to be one of them.

Reply:
I am an anti-fan of that line in general, since it doesn't give the user info anyway on the nature of this particular error. This line has been removed